Files/UserControls/VerticalTabView.xaml

```diff
@@ -430,6 +437,137 @@
             </Setter>
         </Style>
 
+        <Style x:Key="DefaultTabViewListViewStyle" TargetType="primitives:TabViewListView">
+            <Setter Property="VerticalAlignment" Value="Top" />
```

Comment:
Why was it necessary to modify this Style?

Reply:
Yeah, we don't need that, I have removed it.